src/NzbDrone.Core/MediaFiles/EpisodeImport/Specifications/EpisodeTitleSpecification.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.DecisionEngine;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Organizer;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Specifications
+{
+    public class EpisodeTitleSpecification : IImportDecisionEngineSpecification
+    {
+        private readonly IBuildFileNames _buildFileNames;
+        private readonly Logger _logger;
+
+        public RejectionType Type => RejectionType.Temporary;
```

Comment:
You removed Type from the interface, but kept all implementations. Was that intentional?